### PR TITLE
Add French translation #7

### PR DESCRIPTION
**Pull request by Ph0tonic**
Add French translation #7

### CODE REVIEW

File: src/main/resources/assets/pedestalcrafting/lang/fr_fr.lang [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# GUI
+itemGroup.pedestalcrafting=Pedestal Crafting
+
+config.displaySpeed=Affichage de la vitesse
+config.displaySpeed.tooltip=Vitesse à laquel les éléments tournent, s'élèvent et tombent sur les piédestal - rotations/sec
```
Comment (LionelFW) [inline src/main/resources/assets/pedestalcrafting/lang/fr_fr.lang]: "laquel" devrait être "laquelle"

File: src/main/resources/assets/pedestalcrafting/lang/fr_fr.lang [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+# Items
+tile.pedestal.name=Piédestal
+tile.pedestal.diamond.name=Piédestal en diamant
+tile.pedestal.emerald.name=Piédestal en émeraude
+tile.pedestal.glowstone.name=Piédestal en pierre lumineuse
+tile.pedestal.gold.name=Piédestal en or
+tile.pedestal.grass.name=Piédestal en herbe
+tile.pedestal.iron.name=Piédestal en fer
+tile.pedestal.lapis.name=Piédestal en lapis
+tile.pedestal.logacacia.name=Piédestal en bois d'accacias
```
Comment (LionelFW) [inline src/main/resources/assets/pedestalcrafting/lang/fr_fr.lang]: "accacias" devrait être "acacia"

File: src/main/resources/assets/pedestalcrafting/lang/fr_fr.lang [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+tile.pedestal_core.name=Piédestal principal
+tile.pedestal_core.diamond.name=Piédestal principal en diamant
+tile.pedestal_core.emerald.name=Piédestal principal en émeraude
+tile.pedestal_core.glowstone.name=Piédestal principal en pierre lumineuse
+tile.pedestal_core.gold.name=Piédestal principal en or
+tile.pedestal_core.grass.name=Piédestal principal en herbe
+tile.pedestal_core.iron.name=Piédestal principal en fer
+tile.pedestal_core.lapis.name=Piédestal principal en lapis
+tile.pedestal_core.logacacia.name=Piédestal principal en bois d'accacias
```
Comment (LionelFW) [inline src/main/resources/assets/pedestalcrafting/lang/fr_fr.lang]: "accacias" devrait être "acacia"

**Review by LionelFW — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/pedestalcrafting/lang/fr_fr.lang (hunk 1 above)
Comment (Ph0tonic) [reply src/main/resources/assets/pedestalcrafting/lang/fr_fr.lang]: You're right, I corrected it in my last commit

File: src/main/resources/assets/pedestalcrafting/lang/fr_fr.lang (hunk 2 above)
Comment (Ph0tonic) [reply src/main/resources/assets/pedestalcrafting/lang/fr_fr.lang]: You're right, I corrected it in my last commit

File: src/main/resources/assets/pedestalcrafting/lang/fr_fr.lang (hunk 3 above)
Comment (Ph0tonic) [reply src/main/resources/assets/pedestalcrafting/lang/fr_fr.lang]: Idem

File: src/main/resources/assets/pedestalcrafting/lang/fr_fr.lang (hunk 1 above)
Comment (LionelFW) [reply src/main/resources/assets/pedestalcrafting/lang/fr_fr.lang]: Yep, it's all good now, thank you !

Comment (Ph0tonic) [reply src/main/resources/assets/pedestalcrafting/lang/fr_fr.lang]: You're welcome, sorry for the mistakes.

**Pull request closed without merging**